Framework/Core/src/CommonServices.cxx

```diff
@@ -154,24 +154,24 @@ o2::framework::ServiceSpec CommonServices::datatakingContextSpec()
     .configure = noConfiguration(),
     .preProcessing = [](ProcessingContext& processingContext, void* service) {
       auto& context = processingContext.services().get<DataTakingContext>();
-      // Only on the first message
-      if (context.source == OrbitResetTimeSource::Data) {
-        return;
-      }
-      // Only if we do not have already the proper number from CTP
-      if (context.source == OrbitResetTimeSource::CTP) {
-        return;
-      }
-      context.source = OrbitResetTimeSource::Data;
-      context.orbitResetTime = -1;
```

Comment:
Why all this is needed? The `context.source` seems to be nowhere set except at L165 and it is always setting `context.orbitResetTime` to -1.
I think before it was simply a dead code since the OrbitReset object was unconditionally fetched from the server. 
Now, once it is fetched only if the string `context.orbitResetTime` contains `ccdb://CTP/Calib/OrbitReset`, assigning -1 here leads to skipping the OrbitReset fetching and message `[WARN] timestamp 1727494790 for orbit 787466286 and orbit reset time 1657469667 is well behind TF creation time 1657469667915, use the latter`

Reply:
Yes, indeed. I think it's just obsolete code which was used by the old, per device, fetching strategy.

Reply:
Then, shall I simply 
1) remove DataTakingContext.source and line https://github.com/AliceO2Group/AliceO2/blob/3361566da33221b8ea2d7384ed5794a55c4432ca/Framework/Core/src/CommonServices.cxx#L209
2) revert https://github.com/AliceO2Group/AliceO2/blob/3361566da33221b8ea2d7384ed5794a55c4432ca/Framework/Core/src/CCDBHelpers.cxx#L324 to what it was before?
3) convert DataTakingContext.orbitResetTime to long and simply assign to it the value extracted from the CTP object

For the synthetic runs creation time hack the current proxy version is fine, it does not need any accompanying change on the CCDBHelpers or CommonServices side.